### PR TITLE
Add clean stack traces to readme

### PR DESCRIPTION
**Pull request by vdemedes**
Big shout out to @jamestalmage for #418!

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -487,6 +488,14 @@ $ ava --tap | tap-nyan
...
 
 
+### Clean stack traces
+
+AVA automatically removes unrelated lines in stack traces, allowing you to find the source of an error much faster.
+
+<img src="media/stack-traces.png" width="300">
+
+
+
```
Comment (kevva) [inline readme.md]: One too much?

Comment (vdemedes) [inline readme.md]: Yep, fixing, thanks!

**Comment by sindresorhus** [comment]
🎉

**Comment by jamestalmage** [comment]
Should we have explained how to disable that behavior?

```
$ DEBUG=ava ava test/foo.js
```

**Comment by sindresorhus** [comment]
@jamestalmage Nah, we can mention it when we add a `--debug` flag.

**Pull request merged**